netdefine.go

```diff
@@ -87,27 +381,45 @@ func (n *Network) GetNextIp() (string, error) {
 	ipn.Add(ipn, big.NewInt(n.nextIp))
 
 	b := ipn.Bytes()
+	subnetMask := net.IPMask(net.ParseIP(mask))
```

Comment:
whats this doing?

Reply:
this sets the subnet mask for the interface, preferring one defined on a peer, then on its network, then the one encoded in the address space itself. essentially, if you bind to /24 you only have visibility within that /24. this seeks to address that. not sure if it's the best approach